OWNERS_ALIASES

```diff
@@ -44,16 +44,24 @@ aliases:
       - alicefr
       - 0xFelix
       - lyarwood
-  test-reviewers:
+
```

Comment:
How about a sig-api reviewers?  Kubernetes has something similar where a group approves API changes based on design, feature quality, api conventions, ect... 

Also I could see a sig-compute.   The group would handle anything that influences how a vm gets scheduled, e.g. devices, numa, virt-launcher mem/cpu footprint, ect..

Reply:
+1

Reply:
Agreed, thanks !